apps/hash-agents/package.json

```diff
@@ -6,20 +6,20 @@
   "main": "./src/agents.ts",
   "scripts": {
     "build": "yarn exe ./scripts/codegen.ts 'app/agents/**/*.ts'",
-    "build:docker": "DOCKER_BUILDKIT=1 docker build --tag hash-agents --file docker/Dockerfile ../../",
+    "build:docker": "docker buildx build --tag hash-agents --file docker/Dockerfile ../../",
```

Comment:
Is this included by default or do people need to install an extension?

Reply:
https://docs.docker.com/build/install-buildx/ it seems to me that it may not be installed automatically for some folks. Should we take this into consideration? Perhaps adding the requirement to the readme next to the docker/docker compose versions